builtin/providers/aws/resource_aws_spot_fleet_request.go

```diff
@@ -564,8 +583,28 @@ func resourceAwsSpotFleetRequestCreate(d *schema.ResourceData, meta interface{})
 		DryRun:                 aws.Bool(false),
 	}
 
-	log.Printf("[DEBUG] Requesting spot fleet with these opts: %+v", spotFleetOpts)
-	resp, err := conn.RequestSpotFleet(spotFleetOpts)
+	fmt.Printf("[DEBUG] Requesting spot fleet with these opts: %+v", spotFleetOpts)
```

Comment:
Are you sure this goes as a fmt.Printf? I thought `log.` was the right thing so that it responds to TF_DEBUG